src/parameterCfg.m

```diff
@@ -108,12 +126,28 @@
 
 % Path to material library
 if ~isfield(para, 'materialLibraryPath')
-    warning('Environment file path not defined. Using default material library.')
+    warning('Material library path not defined. Using Empty material library.');
     para.materialLibraryPath = 'material_libraries/materialLibraryEmpty.csv';
     cache = fullfile(scenarioNameStr, 'Input/cachedCadOutput.mat');
     if isfile(cache)
         delete(cache)
     end
+else
+    currentPath =pwd;
+    materialLibraryAbsPath = fullfile(currentPath(1:find((pwd==filesep)==1, ...
+        1, 'last')), 'src',para.materialLibraryPath); % For test suite
+    if ~isfile(materialLibraryAbsPath)...
+        || ~isMaterialLibraryFileFormat(para.switchQDModel,...
+        materialLibraryAbsPath,para.materialLibraryPath)
+        warning(strcat('Using Empty material library. Check following issues: ',...
+            ' 1. Material file not exist. ',...
+            ' 2. Material file format not correct.'));
```

Comment:
check spaces

Reply:
warning(['Using Empty material library. Check following issues: ',...
            '1. Material file not exist. ',...
            '2. Material file format not correct.']);

Reply:
this is what you want?
I check both has same spacing

Reply:
I was referring to printed spaces: you have white spaces both at the beginning and at the end of each line, which will be translated into a double space one it gets printed.
To keep the consistency, please keep the white space at the end of the line and remove it from the beginning

Reply:
corrected